src/ai/backend/manager/scheduler/dispatcher.py

```diff
@@ -714,12 +714,16 @@ async def _schedule_single_node_session(
             raise GenericBadRequest(
                 "Cannot assign multiple kernels with different architectures' single node session",
             )
+        if not requested_architectures:
+            raise GenericBadRequest("Requested session has no architecture information")
```

Comment:
`requested_architectures` can be empty when the computing session does not have any sibling computing kernels.
How about checking session's sibling kernels instead of `requested_architectures`?
like this.
```python
if not sess_ctx.kernels:
    raise GenericBadRequest(f"Given session does not have any sibling kernel. (id: {sess_ctx.id}")
```